crates/wash-cli/src/up/mod.rs

```diff
@@ -285,6 +285,26 @@ pub struct WasmcloudOpts {
 }
 
 impl WasmcloudOpts {
+    /// Create a NATS client from the current WasmcloudOpts
+    pub(crate) async fn create_nats_client(&self) -> Result<async_nats_0_33::Client> {
```

Comment:
Should this just be `async_nats` now?

Reply:
We can't change this yet because of `wadm` right -- it's because it uses the older version